keyboards/durgod/k310/keymaps/chimera/keymap.c

```diff
@@ -0,0 +1,367 @@
+    DF_M2WBL               // Change layout from Mac to Windows
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Keymap _WBL: Windows Base Layer (Default Layer)
+     * ,-----------------------------------------------------------.  ,--------------.
+     * |Esc | f1| f2| f3| f4|  | f5| f6| f7| f8|   | f9|f10|f11|f12|  |Prnt|ScLk|Paus|
+     * |-----------------------------------------------------------|  |--------------|  ,---------------.
+     * | ` | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 0 | - | = |Backsp |  | Ins|Home|PgUp|  |NLk| / | * | - |
+     * |-----------------------------------------------------------|  |--------------|  |---------------|
+     * |Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]|    \|  | Del| End|PgDn|  | 7 | 8 | 9 |   |
+     * |-----------------------------------------------------------|  `--------------'  |-----------| + |
+     * |CAPS   |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '| #|Ret |                    | 4 | 5 | 6 |   |
+     * |-----------------------------------------------------------|       ,----.       |---------------|
+     * |Shift| \|  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift     |       | Up |       | 1 | 2 | 3 |   |
+     * |-----------------------------------------------------------|   ,-------------.  |-----------|Ent|
+     * |Ctrl|Gui |Alt |      Space            |Alt |Func |App|Ctrl |   |Lft| Dn |Rig |  |   0   | . |   |
+     * `-----------------------------------------------------------'   `-------------'  `---------------'
+     */
+    [_WBL] = LAYOUT_all( /* Windows Base Layer */
+        KC_ESC,           KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,     KC_PSCR, KC_SLCK, KC_PAUS,
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,    KC_INS,  KC_HOME, KC_PGUP,    KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,    KC_DEL,  KC_END,  KC_PGDN,    KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_NUHS, KC_ENT,                                   KC_P4,   KC_P5,   KC_P6,
+        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,             KC_UP,               KC_P1,   KC_P2,   KC_P3,   KC_PENT,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_WFN,  KC_APP,  KC_RCTL,    KC_LEFT, KC_DOWN, KC_RGHT,    KC_P0,            KC_PDOT
+    ),
+    /* Keymap _WFL: Windows Function Layer
+     * ,-----------------------------------------------------------.  ,--------------.
+     * |   |Play|Stop|Prev|Next| |Mute|Vol+|Vol-|   |  |  |  |  |  |  |    |    |Mute|
```

Comment:
Since these Mute/Vol+/Vol- alternate keymaps seem to be the only difference between this keymap and default_toggle_mac_windows, I'd say lets just combine the two keymaps.

I like your name better; perhaps we just drop default_toggle_mac_windows and keep chimera.

Reply:
I found the name for the keymap `default_toggle_mac_windows` a tad on the long side.  I'm game to do whatever you recommend if it makes it easier to get everything merged to `master`.  The name "chimera" seemed appropriate since it combines macOS and Windows functionality (and as I have been running these K310 builds on all the K320s and K310s I maintain because it was simpler).

I had intended to make some slightly more radical changes to a keymap after everything else was merged and after my Ploopy Nano trackballs arrive so I was selfishly hoping I would have a suitable keymap for my changes that wouldn't be too disruptive to others (hence the new keymap).  I'm pretty sure I can probably do some cool layer tricks to allow my stuff to peacefully-coexist with an existing keymap.

I'm quite happy to share and am looking forward to the fun of collaborating on these improvements.  The Ploopy Nano trackballs don't have buttons so I'll likely be adding some more pointing device tricks with a strong focus on Linux/Unix usability.